ocs_ci/utility/azure_utils.py

```diff
@@ -0,0 +1,95 @@
+import logging
+
+from azure.common.credentials import ServicePrincipalCredentials
+from azure.mgmt.resource import ResourceManagementClient
+from azure.mgmt.network import NetworkManagementClient
+from azure.mgmt.compute import ComputeManagementClient
+from azure.mgmt.compute.models import DiskCreateOption
```

Comment:
Since the azure APIs are from `azure` package, named the file as **azure_utils.py** instead of **azure.py** to avoid import ambiguity.